src/main/java/org/cyclopsgroup/jmxterm/boot/CliMain.java

```diff
@@ -80,6 +80,7 @@ int execute(String[] args) throws Exception {
         } else {
           LineReaderImpl consoleReader = (LineReaderImpl) LineReaderBuilder.builder().build();
           File historyFile = new File(System.getProperty("user.home"), ".jmxterm_history");
+          output.printMessage("Delete " + historyFile.getAbsolutePath() + " if you encounter error right after launching me.");
```

Comment:
When I was testing my change, I have such exception by reading from history. adding this message to tell user how to solve such issue manually.